src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -2914,12 +3039,26 @@ bool hasImplicitConversion(TypeSymbol source, TypeSymbol destination, ref Compou
                 Conversions.ClassifyImplicitConversionFromType(source, destination, ref useSiteInfo).IsImplicit;
         }
 
-        private bool IsBetterParamsCollectionType(TypeSymbol t1, TypeSymbol t2, ref CompoundUseSiteInfo<AssemblySymbol> useSiteInfo)
+        private BetterResult IsBetterParamsCollectionType(TypeSymbol t1, TypeSymbol t2, ref CompoundUseSiteInfo<AssemblySymbol> useSiteInfo)
```

Comment:
>IsBetterParamsCollectionType [](http://example.com/codeflow?start=29&length=28)

Perhaps rename to `BetterParamsCollectionType` since this no longer returns a `bool`.